docs/docsite/rst/dev_guide/developing_collections_shared.rst

```diff
@@ -65,6 +65,21 @@ By wrapping the import statement in a ``try`` block, the payload builder will om
 
 The optional import behavior also applies to module_utils imported from collections.
 
+.. _collection_dependencies:
+
+Adding collection dependencies
+==============================
+
+If your collection depends on features and functionality from another collection, ????
+
+You can use git repositories for collection dependencies during local development and testing. For example:
+
+.. code-block:: yaml
+
+   dependencies: {'git@github.com:organization/repo_name.git': 'devel'}
+
+Do not use git repositories as dependencies for published collections. Dependencies for published collections must be other published collections.
```

Comment:
Should this be a NOTE?

Reply:
I made it a warning. See 2b84951fc61422551f159ccf5df686beff5fb23e.